backend/core/Extensions/GeometryExtensions.cs

```diff
@@ -0,0 +1,80 @@
+using NetTopologySuite.Geometries;
+
+namespace Pims.Core.Extensions
+{
+    /// <summary>
+    /// GeometryExtensions static class, provides extension methods for geometry objects.
+    /// </summary>
+    public static class GeometryExtensions
+    {
+        /// <summary>
+        /// Convert an Envelope to a Polygon.
+        /// </summary>
+        /// <param name="envelope"></param>
+        /// <returns></returns>
+        public static Polygon ToPolygon(this Envelope envelope)
```

Comment:
Centralize this function so that all the queries that use it can be updated in one place.